From b6241002039118d6bea78f50f8f19195b41ab602 Mon Sep 17 00:00:00 2001
From: Yan Zhao <yan.y.zhao@intel.com>
Date: Tue, 7 May 2019 22:16:33 -0400
Subject: drm/i915/gvt: add 0x4dfc to gen9 save-restore list
Git-commit: b6241002039118d6bea78f50f8f19195b41ab602
Patch-mainline: v5.2-rc2
References: bsc#1113722

0x4dfc is in-context mmio for gen9+, but each vm have different settings
need to add it to save-restore list along with other trtt registers

Fixes: 178657139307 ("drm/i915/gvt: vGPU context switch")
Acked-by: Zhenyu Wang <zhenyuw@linux.intel.com>
Signed-off-by: Yan Zhao <yan.y.zhao@intel.com>
Signed-off-by: Zhenyu Wang <zhenyuw@linux.intel.com>
Acked-by: Thomas Zimmermann <tzimmermann@suse.de>
---
 drivers/gpu/drm/i915/gvt/mmio_context.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/gpu/drm/i915/gvt/mmio_context.c
+++ b/drivers/gpu/drm/i915/gvt/mmio_context.c
@@ -125,6 +125,7 @@ static struct engine_mmio gen9_engine_mm
 	{RCS, TRINVTILEDETCT, 0, true}, /* 0x4dec */
 	{RCS, TRVADR, 0, true}, /* 0x4df0 */
 	{RCS, TRTTE, 0, true}, /* 0x4df4 */
+	{RCS, _MMIO(0x4dfc), 0, true},
 
 	{BCS, RING_GFX_MODE(BLT_RING_BASE), 0xffff, false}, /* 0x2229c */
 	{BCS, RING_MI_MODE(BLT_RING_BASE), 0xffff, false}, /* 0x2209c */
